provider/openstack/provider.go

```diff
@@ -1338,6 +1338,19 @@ func (e *Environ) startInstance(
 	}, nil
 }
 
+func (e *Environ) userFriendlyInvalidNetworkError(err error) error {
+	msg := fmt.Sprintf("%s\n\t%s\n\t%s", err,
+		"This error was caused by juju attempting to create an OpenStack instance with no network defined.",
+		"No network has been configured.")
+	networks, err := e.networking.FindNetworks(true)
+	if err != nil {
+		msg = fmt.Sprintf("%s\n\t%s\n\t\t%s", msg, "Error attempting to find internal networks:", err.Error())
+	} else {
+		msg = fmt.Sprintf("%s %s\n\t\t%q", msg, "The following internal networks are available: ", strings.Join(networks.Values(), ", "))
```

Comment:
```suggestion
		msg += fmt.Sprintf("\nThe following internal networks are available:\n  %q", strings.Join(networks.Values(), ", "))
```

AFAICT the client uses double-spaces instead of tabs for indentation.

Reply:
we're inconsistent with the spacing.  keeping inline with what the rest of the openstack provider does at this time.